### ISSUE TITLE
Cannot import dropboxdrivefs after upgrade it from 1.3.1 to 1.4.0

### ISSUE THREAD

**Issue by jeffrey82221** [fsspec/dropboxdrivefs#17]
Hello, 

After upgrading to 1.4.0, the package cannot be found anymore in python. What might be the problem? 

I am in python3.10 environment. 

Originally, I install dropboxdrivefs 1.3.1 and the import is fine:

```
pip install dropboxdrivefs==1.3.1
```
>>
```
Installing collected packages: dropboxdrivefs
  Attempting uninstall: dropboxdrivefs
    Found existing installation: dropboxdrivefs 1.4.0
    Uninstalling dropboxdrivefs-1.4.0:
      Successfully uninstalled dropboxdrivefs-1.4.0
Successfully installed dropboxdrivefs-1.3.1
```

```python
import dropboxdrivefs
```

But after updating it to 1.4.1, it cannot be imported anymore:

```
pip install dropboxdrivefs==1.4.1
```

```
Installing collected packages: dropboxdrivefs
  Attempting uninstall: dropboxdrivefs
    Found existing installation: dropboxdrivefs 1.3.1
    Uninstalling dropboxdrivefs-1.3.1:
      Successfully uninstalled dropboxdrivefs-1.3.1
Successfully installed dropboxdrivefs-1.4.0
```

```
Python 3.10.6 (main, Aug 30 2022, 04:58:14) [Clang 13.1.6 (clang-1316.0.21.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import dropboxdrivefs
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'dropboxdrivefs'
```

What is happening here?

**Comment by MarineChap** [comment]
Hello, 
I cannot reproduce the problem. Are you working in an environment (conda, other?) ?

**Comment by chandrasekharan-zipstack** [comment]
I notice the same issue - also the release isn't updated in Github, not sure if its related in any way though
![image](https://github.com/fsspec/dropboxdrivefs/assets/117059509/4d17ebaa-55ef-4d4e-8022-77bf53aa1956)

FYI when I download the `tar.gz` for `1.3.1` I'm able to view the source code however I don't see it with `1.4.0`

**Comment by MarineChap** [comment]
Indeed I can see the issue with the tar.gz ... 

Ok it is fixed now with the version 1.4.1. Thanks for the feedback!
